department-of-veterans-affairs/caseflow
# adding temporary under_constructions task urls for merge and split tasks.

Resolves [Users that have access to Correspondence cases page can access correspondence details page when selecting veterans not on the completed tab](https://jira.devops.va.gov/browse/APPEALS-52202)

# Description
Fixed bug where users would be routed to the correspondence details page for split and merge package tasks

## Acceptance Criteria
- [ ] Users are not routed to correspondence details page from the Action Required tba or the Pending tab.

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below

1. Log into caseflow as INBOUND_OPS_TEAM_ADMIN_USER at 101 or INBOUND_OPS_TEAM_SUPERUSER at 101
2. Navigate to the Correspondence cases page
3. On the Correspondence cases page, click on the Action Required tab, then click on a case link for a split package or merge package task.
4. Expected result is for these cases to be routed to /under_construction. Previously, links were routing to the correspondence details page if they were part of a merge package or split package task 
5. Return to the Correspondence cases page, click on the Pending tab, then click on a case link for a split package or merge package task. These cases also should now route to /under_construction instead of the previous result routing to the correspondence details page.
